src/scripts/install-aws-iam-authenticator.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 DOWNLOAD_URL="https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v${VERSION}/${FILENAME}_${VERSION}_${PLATFORM}_amd64"
 
-printf "Downloading version ${VERSION}\n"
+echo "Downloading version ${VERSION}"
```

Comment:
Printf can be used here, you just need to put the env var in differently.
```suggestion
printf "Downloading version %s\n"  "${VERSION}"
```